### PR TITLE
Fixing links in RELEASE_MANAGEMENT.md, TRANSLATIONS.md and OBJC_STYLE.md.

### PR DESCRIPTION
**Pull request by Kushagra-Bhagat**
Signed-off-by: Kushagra Bhagat <bhagatkushagra@gmail.com>

### CODE REVIEW

File: docs/TRANSLATIONS.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Main:
 Additional:
 - Text-to-speech strings for navigation: [`data/strings/sound.txt`](../data/strings/sound.txt)
 
-- Android stores description: [`android/src/google/play/listings/`](../android/src/google/play/listings/)
+- Android stores description: [`android/src/google/play/listings/`](https://github.com/Kushagra-Bhagat/organicmaps/tree/master/android/src/google/play)
```
Comment (Altonss) [inline docs/TRANSLATIONS.md:28]:
```suggestion
- Android stores description: [`android/src/google/play/listings/`](https://github.com/organicmaps/organicmaps/tree/master/android/src/google/play)
```

File: docs/RELEASE_MANAGEMENT.md [hunk 2]
```diff
@@ -2,13 +2,9 @@
...
 
-### Upload metadata to AppStore
+### Upload metadata and screenshots to AppStore
 
-Use [GitHub Actions](../.github/workflows/ios-release-metadata.yaml).
-
-### Upload screenshots to AppStore
-
-Use [GitHub Actions](../.github/workflows/ios-release-screenshots.yaml).
+Use [GitHub Actions](https://github.com/Kushagra-Bhagat/organicmaps/blob/master/.github/workflows/ios-release.yaml).
```
Comment (Altonss) [inline docs/RELEASE_MANAGEMENT.md:7]:
```suggestion
Use [GitHub Actions](https://github.com/organicmaps/organicmaps/blob/master/.github/workflows/ios-release.yaml).
```

File: docs/TRANSLATIONS.md (hunk 1 above)
Comment (Kushagra-Bhagat) [reply docs/TRANSLATIONS.md:28]: ok will do

File: docs/OBJC_STYLE.md [hunk 3]
```diff
@@ -331,7 +331,7 @@ Init methods should follow the convention provided by Apple's generated code tem
 }
 ```
 
-See [Class Constructor Methods](#class-constructor-methods) for link to article on instancetype.
+See [Class Constructor Methods](https://github.com/organicmaps/organicmaps/blob/master/docs/OBJC_STYLE.md#class-constructor-methods) for link to article on instancetype.
```
[the remark this answers is not in the document]
Comment (Kushagra-Bhagat) [reply docs/OBJC_STYLE.md:334]: The links before were redirecting to the beginning of the documentation.

**Comment by rtsisyk** [comment]
Closed because of inactivity. Please feel free to re-open.

**Pull request closed without merging**